jest.config.js

```diff
@@ -8,9 +8,9 @@ module.exports = {
   coverageThreshold: {
     global: {
       functions: 100,
-      lines: 100,
+      lines: 95,
       branches: 100,
-      statements: 100,
+      statements: 95,
```

Comment:
this PR introduced a regression  in the  src/type/index.ts that is tested by tsd but it can't be seen in the coverage